### ISSUE TITLE
EW update?

### ISSUE THREAD

**Issue by Dajova** [Arrenton/MouseTargetTooltip#2]
It's been a few weeks now since the EW launch. Any plans to update this plugin to support it?

**Comment by Arrenton** [comment]
Yeah, if I could play the game lol. Queues are horrible at times I can play.
